slf4j-api/src/main/java/org/slf4j/MDC.java

```diff
@@ -235,6 +235,9 @@ public static void setContextMap(Map<String, String> contextMap) {
         if (mdcAdapter == null) {
             throw new IllegalStateException("MDCAdapter cannot be null. See also " + NULL_MDCA_URL);
         }
+        if (contextMap == null) {
+            contextMap = new HashMap<String, String>();
```

Comment:
The build in this branch is broken because `HashMap` is not imported in this class.
We need to add `import` to the head of this file, or use FQCN like below:

```suggestion
            contextMap = new java.util.HashMap<String, String>();
```